### PR TITLE
unit tests

### PR DESCRIPTION
**Pull request by Amialc**
Link accounts test doesn't feel to be right. @christiannwamba waiting for your feedback

### CODE REVIEW

**Comment by christiannwamba** [comment]
Good job @Amialc 

I just reviewed and it is awesome. Was wondering if it is possible to stub the callbacks rather than calling them directly. What do you think?

**Comment by christiannwamba** [comment]
I will go ahead and merge this and then we work on improvement if there is room for that

File: test/unit/services/auth0.service.test.js [hunk 1]
```diff
@@ -77,6 +95,24 @@ describe('Auth0 Angular', function () {
...
+
+        beforeEach(inject(function (_auth_, _$rootScope_, _$timeout_) {
+            auth = _auth_;
+            $rootScope = _$rootScope_;
+            $timeout = _$timeout_;
+        }));
+
+        it('link', function() {
+            auth.linkAccount('token', {}, {connection:'facebook'},
+                function() {
```
Comment (christiannwamba) [inline test/unit/services/auth0.service.test.js]: Was wondering if it is possible to stub the callbacks rather than calling them directly.

Comment (Amialc) [inline test/unit/services/auth0.service.test.js]: Do you mean `sinon.stub()` without any parameters?

Comment (christiannwamba) [inline test/unit/services/auth0.service.test.js]: Yeah something like that. I am not really good with testing but something like that is what I mean

**Pull request merged**